math/minuit2/StandAlone.cmake

```diff
@@ -1,4 +1,8 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.1...3.11)
```

Comment:
Is this syntax documented anywhere? I cannot find it. Which warnings do you see?

Reply:
Yes, [here](https://cmake.org/cmake/help/git-master/command/cmake_minimum_required.html#command:cmake_minimum_required). The warnings are about CMake using old policy settings. The recommended way is to avoid setting specific policies to NEW, and instead set the oldest version of CMake supported and the newest version of CMake tested (3.11, in this case). The if statement below is used if CMake < 3.12.

Reply:
There's also a discussion somewhere about why that syntax was chosen, due to it not causing an error with old versions of CMake.

Reply:
Ah, ok, if it does not cause errors with earlier versions, then that's fine. I usually look at the latest release docs, so I missed it... Thanks for the clarification.